modules/core/src/test/scala/org/tessellation/http/routes/TrustRoutesSuite.scala

```diff
@@ -63,4 +85,67 @@ object TrustRoutesSuite extends HttpSuite {
       }
       .unsafeRunSync()
```

Comment:
Using `unsafeRunSync` should be a red flag that something's not right because the `test` method is already doing that for us. Chances are your `yield` is returning an `IO[...]`

Reply:
Fixed.